integration/shared/isolated/delete_orphaned_routes_command_test.go

```diff
@@ -90,10 +87,6 @@ var _ = Describe("delete-orphaned-routes command", func() {
 			It("deletes only the orphaned routes", func() {
 				Eventually(helpers.CF("delete-orphaned-routes", "-f")).Should(SatisfyAll(
 					Exit(0),
-					Say("Getting routes as"),
-					Say(fmt.Sprintf("Deleting route orphan-1.%s/path-1...", domainName)),
-					Say(fmt.Sprintf("Deleting route orphan-2.%s/path-2...", domainName)),
-					Not(Say(fmt.Sprintf("Deleting route bound-1.%s/path-3...", domainName))),
```

Comment:
Please add a call to `cf routes` and assert that only the bound route is still present - otherwise, this test doesn't really cover what its description says it's covering.